### DISCUSSION TITLE
Link Automatically removing when I remove element connected to the link

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2116] [Q&A]
### Introduction

![1122](https://user-images.githubusercontent.com/117301232/230011389-12f4c2a0-41b3-4b36-9e63-f00e764e15fd.png)

second image after deleting the first element

![112233](https://user-images.githubusercontent.com/117301232/230012020-0d7b05c7-9c33-4651-9d41-eb212c29bc63.png)

### Steps to reproduce

above image, I connected two elements with the help of a link, When I remove one element the Link also automatically removes. Is there any way to avoid that

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
As per [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Element.prototype.remove), you can use `disconnectLinks` option.

```js
element.remove({ disconnectLinks: true });
```

*Note that this changes all attached links to the position `{ x:0, y:0 }`.*

---

Another option is to reconnect the links first and then remove the element.

If you want the links to stay exactly as they were, you need to ask `LinkView` for the original exact connection point. Here's an [example](https://jsfiddle.net/kumilingus/f8acxypq/).

**Reply by reshin-pulapaddy** [reply]
Thank you @kumilingus
